src/index.js

```diff
@@ -117,12 +113,14 @@ export const getTargets = (targets = {}) => {
 
     delete targetOps.electron;
   }
+  browserslist.defaults = objectToBrowserslist(targetOps);
+
+  const browsersQuery = targetOps.browsers;
+  const browserslistOpts = {path: opts.path};
```

Comment:
@existentialism I was thinking about this but decided we could have common `path` option for all things. For example, while searching package.json for `engines` in the future. And apparently, it would be the same path for package.json in this case.
I agree that we should pay attention to this, but ideally try not to spread new options, but to unify it and make it works in chime.
If we really could have problems with a common path, `browserslistConfigPath ` is good. Or `browserslistPath`, because it's also necessary for `package.json` searching.